migrations/mystery_items.js

```diff
@@ -27,6 +27,7 @@ if (_id) {
   db.users.update({_id:_id}, update);
 } else {
   // multiple (once @ start of event)
-  db.users.update({'purchased.plan.customerId':{$ne:null}}, update, {multi:true});
-}
-
+  db.users.find({'purchased.plan.customerId':{$ne:null}}).forEach(function(user) {
```

Comment:
Is the find and forEach necessary? Can the update call not be done all at once? Is there a performative difference?